purchases/src/main/java/com/revenuecat/purchases/customercenter/CustomerCenterManagementOption.kt

```diff
@@ -0,0 +1,24 @@
+package com.revenuecat.purchases.customercenter
+
+import android.net.Uri
+
+/**
+ * Sealed interface representing different customer center management options.
+ */
+sealed interface CustomerCenterManagementOption {
```

Comment:
Just making sure that using `sealed` is a conscious decision? What are we going to do when adding an `*Option`? The alternative would be to drop the `sealed`:
```suggestion
interface CustomerCenterManagementOption {
```

Reply:
oh right... I added it so it could be used in `when` but yeah, same problems as with the enums when adding a new one. Good point

Reply:
A regular `interface` can also be used in a `when`, but the `else` is mandatory in that case.